.github/workflows/ansible-test.yml

```diff
@@ -1,8 +1,9 @@
 # README FIRST
-# 1. replace "NAMESPACE" and "COLLECTION_NAME" with the correct name in the env section (e.g. with 'community' and 'mycollection')
-# 2. If you don't have unit tests remove that section
-# 3. If your collection depends on other collections ensure they are installed, see "Install collection dependencies"
-# If you need help please ask in #ansible-community on the Libera.chat IRC network
+# 1. If you don't have unit tests, remove that section.
+# 2. If your collection depends on other collections ensure they are installed,
+#    see "Install collection dependencies".
+# 3. If you need help please ask in #ansible-community on the Libera.chat IRC
+#    network.
```

Comment:
I think there should also be a link to https://github.com/ansible-community/ansible-test-gh-action as a reference on what the shared workflow exactly supports, as not all options are listed here.

Reply:
Fair enough.

Reply:
Done!